libs/service/api_service.cpp

```diff
@@ -86,6 +91,12 @@ _build_helper_id_to_address_map(
         }
         instruction.imm = helper_id_mapping[instruction.imm];
     }
+    for (auto& [old_helper_id, new_helper_id] : helper_id_mapping) {
+        if (get_helper_prototype_windows(old_helper_id).return_type != EBPF_RETURN_TYPE_VOID)
```

Comment:
is this the criteria for detecting tail call helper?

Reply:
I dont understand. The prototype for tail call is:
`
     "bpf_tail_call",
     EBPF_RETURN_TYPE_INTEGER,
     {EBPF_ARGUMENT_TYPE_PTR_TO_CTX, EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_ANYTHING}
`